app/src/main/java/com/github/geohunt/app/ui/DatabaseUtils.kt

```diff
@@ -0,0 +1,20 @@
+package com.github.geohunt.app.ui
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import com.github.geohunt.app.authentication.Authenticator
+import com.github.geohunt.app.model.database.api.Database
+import com.github.geohunt.app.model.database.api.LoggedUserContext
+
+/**
+ * Helper to use the [Database.logged] function with composable callback
+ *
+ * @param callback the composable callback to be used within the logged context
+ */
+@Composable
+fun Database.Logged(callback : @Composable LoggedUserContext.() -> Unit) {

```

Comment:
The function name `Logged` is not very descriptive. Consider renaming it to something more informative, such as `WithLoggedUserContext`.